person.rb

```diff
@@ -11,6 +12,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] As part of the requirements `A method for adding rentals should be present ,  kindly add that method to the person class 🙏